backend/compact-connect/lambdas/python/data-events/tests/function/test_encumbrance_events.py

```
@@ -0,0 +1,706 @@
+                'licenseJurisdiction': DEFAULT_LICENSE_JURISDICTION,
+                'licenseTypeAbbreviation': DEFAULT_LICENSE_TYPE_ABBREVIATION,
+                'encumberedStatus': 'encumbered',  # Already encumbered
+                'jurisdiction': 'ky',  # Different jurisdiction to distinguish
+            }
+        )
+
+        message = self._generate_license_encumbrance_message()
+        event = self._create_sqs_event(message)
+
+        # Execute the handler
+        license_encumbrance_listener(event, self.mock_context)
```

Pylint (3.3.7)

[error] Too many positional arguments for function call

(E1121)